service-front/module/Application/view/application/pages/cp/add_lpa.twig

```diff
@@ -52,12 +54,6 @@
         example: M-0000-0000-0000
     </p>
     <div class="govuk-form-group {% if form.get('lpa').messages %}govuk-form-group--error{% endif %}">
-        {% if form.get('lpa').messages %}
-            <p id="lpa-error" class="govuk-error-message">
-                <span class="govuk-visually-hidden">Error:</span>
-                {{ form.get('lpa').messages | join(', ', ' and ') }}
-            </p>
-        {% endif %}
```

Comment:
Looks like you've removed this, but assigned something to `lpa` error message in CPFlowController ("Not a valid LPA number. Enter an LPA number to continue."). If we're setting a message, don't we want to display it?

Reply:
On the add_lpa screen, the errors are displayed in a red box at the top of the page. The Figma doesn;t specify whether the forma validation error should also appear next to the input, but when I allowed the error to show twice it made the page look a bit messy

Reply:
Ah I see what you mean